.github/workflows/unit-tests.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        version: [18]
+        version: [20]
```

Comment:
We no longer run a matrix of Node versions. Maybe we should? If not, maybe we should strip it all out of here and pull in the Node version from an environment variable?